storage/google/cloud/storage/blob.py

```diff
@@ -432,8 +432,10 @@ def download_to_filename(self, filename, client=None):
         with open(filename, 'wb') as file_obj:
             self.download_to_file(file_obj, client=client)
 
-        mtime = time.mktime(self.updated.timetuple())
-        os.utime(file_obj.name, (mtime, mtime))
+        updated = self.updated
```

Comment:
No change needed, but wondering why you've assigned this to a temporary variable instead of just referencing `self.updated`?